src/components/Questions/styles.tsx

```diff
@@ -18,3 +16,7 @@ export const ButtonView = styled.View`
   flex-direction: row;
   justify-content: center;
 `;
+
+export const Container = styled.ScrollView`
```

Comment:
[PROBABLY WRONG, MIGHT NOT REQUIRE ACTION] This padding might be acting on the scroll bar, not necessarily the page view itself? Might have to do something similar with PageContainer and InnerScroll 
<img width="368" alt="Screen Shot 2021-12-05 at 12 27 13 AM" src="https://user-images.githubusercontent.com/63089231/144739310-f9968325-51d6-4bb8-9c07-667c74e2f25f.png" >

Reply:
This error doesn't show up for GeneralQuestions though, so I'm a bit confused why this is happening. If I keep submitting the form (below is the 3rd-4th try) the scroll bar keeps moving further center. WHAT IS THIS BUG. Probably not your doing, but documenting the problem here: 
<img width="392" alt="Screen Shot 2021-12-05 at 12 31 13 AM" src="https://user-images.githubusercontent.com/63089231/144739444-f66a8347-779f-4ab4-8e09-182072190665.png">

Reply:
can you inspect element at all? this doesn't show up on web